third_party/amd/include/Dialect/TritonAMDGPU/IR/TritonAMDGPUOps.td

```diff
@@ -519,6 +519,58 @@ def TTG_UpcastMXFPOp : TT_AMDGPU_Op<"upcast_mxfp", [Pure]> {
   }];
 }
 
+//===----------------------------------------------------------------------===//
+// MaskedLoadOp
+//===----------------------------------------------------------------------===//
+def MaskedLoadOp : TT_AMDGPU_Op<"masked_load", []> {
+  let summary = "Masked load operation";
+  let description = [{
+    Load operation with masked support. When the mask is existing, loads from the given pointer. WOrks with LLVM types.
```

Comment:
".. with masking support.. If the mask is true, .. Works with LLVM types as a utility op for making LLVM conversion easier.". Similarly for the store op.